model/src/main/kotlin/CuratedPackage.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+/**
+ * A [Package] plus the [PackageCurationResult]s to be able to trace how the raw metadata of the package was modified
+ * by applying [PackageCuration]s.
+ */
+data class CuratedPackage(
+        val pkg: Package,
```

Comment:
I'd slightly prefer to (de-)serialize this as "package", as we also say "packages" in the file output.

Reply:
Done.